src/CalcManager/CEngine/scicomm.cpp

```diff
@@ -790,6 +832,40 @@ void CCalcEngine::ProcessCommandWorker(OpCode wParam)
         break;
 
     case IDC_PNT:
+
+        // Check if the last command was a closing parenthesis
```

Comment:
This part is the same as the change above, suggest creating a method for this change.

Reply:
It's a nice-to-have work.
I think we can let this change get in to fix the bug and then do some refactoring for it later.